src/storage-client/src/client.rs

```diff
@@ -303,6 +305,42 @@ pub struct SourceStatisticsUpdate {
     pub bytes_received: u64,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct SinkStatisticsUpdate {
+    pub id: GlobalId,
+    pub worker_id: usize,
+    pub messages_staged: u64,
```

Comment:
this is very nit-y: but I'd also order them differently (see my comment above)

Reply:
I agree, all the orderings should match across the entire codebase~